extensions/wikia/LookupUser/templates/contribution.table.tmpl.php

```diff
@@ -236,10 +242,12 @@
 	<tfoot>
 		<tr>
 			<th width="2%">#</th>
-			<th width="25%"><?= wfMessage( 'lookupuser-table-title' )->escaped() ?></th>
-			<th width="20%"><?= wfMessage( 'lookupuser-table-url' )->escaped() ?></th>
+			<th width="20%"><?= wfMessage( 'lookupuser-table-title' )->escaped() ?></th>
+			<th width="15%"><?= wfMessage( 'lookupuser-table-url' )->escaped() ?></th>
 			<th width="20%"><?= wfMessage( 'lookupuser-table-lastedited' )->escaped() ?></th>
 			<th width="15%"><?= wfMessage( 'lookupuser-table-editcount' )->escaped() ?></th>
+			<th width="5%">Posts</th>
```

Comment:
There're so many columns in the table that it can go over the screen width, so I'm trying to keep it short. Especially since only staff use this page.